broker/lib/tags.py

```diff
@@ -1,11 +1,85 @@
-def add_tag(tags, tag_key: str, tag_value: str):
+import typing
+
+from broker.extensions import config
+from broker.lib.cf import CFAPIClient
+
+from enum import Enum
+from openbrokerapi.service_broker import (
+    ProvisionDetails,
+    ServicePlan,
+    Service,
+)
+
+cf_api_client = CFAPIClient()
+
+
+class Action(Enum):
+    CREATE = "Create"
+    UPDATE = "Update"
+
+
+class Tag(typing.TypedDict):
+    Key: str
+    Value: str
+
+
+def add_tag(tags: list[Tag], tag_key: str, tag_value: str) -> list[Tag]:
     if not tags:
-        tags = {}
-    tags["Items"] = tags.get("Items", [])
-    tags["Items"].append(
+        tags = []
+    tags.append(
```

Comment:
is it OK if we get multiple `tag`s with the same `Key`?

Reply:
Good point. f094f49e1f147646b17794a528e5c7d6986bc33a should address it